word_guess.rb

```diff
@@ -0,0 +1,157 @@
+# pair_programming https://github.com/Ada-C7/Word-Guess
+# Elizabeth and Tamiko
+
+require 'colorize'
+
+# new class for word game
+class WordGame
+
+  def initialize
+    @win = false
+    @word = ["PIZZA", "JUMPY", "HELLO", "WORLD", "QUACK", "TAXES"].sample
+    @all_guesses = []
+    @duck_count = 5
+    @open_game = open_game
+
+    # first turn
+    @guess = guess #for test
+    @word_with_blanks = word_with_blanks #for test
+    @score = score
+    @print_ducks = print_ducks
+
+    # game play
+    until (@win || lose?)
+        guess
+        word_with_blanks
+        score #after_turn
+        print_ducks
+    end
+  end
+
+  def open_game
+    puts "\nWelcome to the DUCKS IN A ROW!"
+    puts "\nYou have 5 guesses to guess the word or the ducklings will swim away."
+    print_ducks
+  end
+
+  def guess
+    print "\nWhich letter would you like to guess? > "
+    @guess = gets.chomp.upcase
+    # validate guess
+    until ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "O", "P", "Q", "R",
+    "S", "T", "U", "V", "W", "X", "Y", "Z"].include?(@guess)
+      print "\n  Quack! Give me a letter! > "
+      @guess = gets.chomp.upcase
+    end
+
+    @all_guesses << @guess
+    return @guess
+  end
+
+  def word_with_blanks
```

Comment:
I also really like this method - it does a good job of encapsulating one bit of functionality, helping to simplify the rest of your program.